boards/beagle/beagleplay_cc1352/doc/index.rst

```diff
@@ -0,0 +1,193 @@
+| DIO8  | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO9  | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO10 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO11 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO12 | CC1352_RX    | AM62 UART6_TXD                      |
++-------+--------------+-------------------------------------+
+| DIO13 | CC1352_TX    | AM62 UART6_RXD                      |
++-------+--------------+-------------------------------------+
+| DIO14 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO15 | CC1352_BOOT  | AM62 GPIO0_13                       |
++-------+--------------+-------------------------------------+
+| DIO16 | CC1352_TDO   | TAG-CONNECT TDO                     |
++-------+--------------+-------------------------------------+
+| DIO17 | CC1352_TDI   | TAG-CONNECT TDI                     |
++-------+--------------+-------------------------------------+
+| DIO18 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO19 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO20 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO21 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO22 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO23 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO24 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO25 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO26 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO27 | LED1         | CC1352_LED1 yellow LED9             |
++-------+--------------+-------------------------------------+
+| DIO28 | LED2         | CC1352_LED2 yellow LED8             |
++-------+--------------+-------------------------------------+
+| DIO29 | RF_PA        | Antenna mux PA enable               |
++-------+--------------+-------------------------------------+
+| DIO30 | RF_SUB1G     | Antenna mux SubG enable             |
++-------+--------------+-------------------------------------+
+
```

Comment:
Note that the antenna mux is only used for antenna J9, the SubG antenna, and muxes between the SubG RX/TX and PA TX paths. J12 is always connected to the 2.4GHz RX/TX path. Internally, I believe it might be possible to use the PA TX path for 2.4GHz, but the design intention is to use J9 for SubG RX and PA TX for SubG signals.

There might be a clearer way to explain that, but as I read the notes, the antenna mux almost implies it is the only antenna.

For simplicity, maybe just call it the "SubG/PA Antenna Mux"?